Web/Design a Library Management System/MemberService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Design_a_Library_Management_System.Model;
+using DesignLibraryManagementSystem.Models;
+using Library;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Design_a_Library_Management_System
+{
+    public class MemberService : ControllerBase, IMemberService
+    {
+        private readonly LibraryContext _context;
+
+        public MemberService(LibraryContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<ActionResult<IEnumerable<LibraryMemberDTO>>> GetLibraryMembers()
```

Comment:
```suggestion
        public async Task<ActionResult<IEnumerable<LibraryMemberDTO>>> GetLibraryMembers()
```
```suggestion
        public async Task<IEnumerable<LibraryMemberDTO>> GetLibraryMembers()
```

ActionResult is needed only in a controller